src/vm/jitinterface.h

```diff
@@ -22,6 +22,14 @@
 #define MAX_UNCHECKED_OFFSET_FOR_NULL_OBJECT ((GetOsPageSize() / 2) - 1)
 #endif // !FEATURE_PAL
 
+
+enum StompWriteBarrierXXXCompletion
+{
```

Comment:
can we call this something else? something like StompWriteBarrierCompletionAction?